src/component/Component.tsx

```diff
@@ -0,0 +1,21 @@
+import { Component, ReactNode } from 'react';
+
+export class Key extends Component {
```

Comment:
Consider extending React.Component with appropriate TypeScript generic, such as React.Component<{}, StateType> to ensure type safety.